vpr/src/route/route_timing.cpp

```diff
@@ -1385,27 +1393,29 @@ t_timing_driven_node_costs evaluate_timing_driven_node_costs(const t_timing_driv
             new_costs.backward_cost += bend_cost; //Bend cost
         }
     }
-
+    float cost_from_budgets = 0;
```

Comment:
This variable appears to be unused; it should probably be removed if not needed (currently generates a compilation warning).